model/schema/allele.yaml

```diff
@@ -61,7 +61,7 @@ classes:
       - allele_secondary_ids
       - allele_nomenclature_events
       - allele_notes
-      - allele_synonyms
+      # - allele_synonyms    # Allele inherits synonyms slot from GenomicEntity
```

Comment:
From the Java perspective I think we would need to keep the `allele_synonyms` slot (but change the range to `AlleleSynonymSlotAnnotation`) and remove the `synonyms` slot from `BiologicalEntity`, rather than specifying the range in the slot_usage.  This is because we would need to specify the 'back reference' to the AlleleSynonymSlotAnnotation table in the Allele class.

Reply:
Thanks Mark. I'll restore `allele_synonyms` and remove the generic `synonyms` slot. It seems like we should also have class specific slots like `allele_symbol` or `gene_full_name` rather than generic `symbol` and `full_name` slots in the `core.yaml`. Thoughts?